scripts/main.js

```diff
@@ -0,0 +1,8 @@
+import {
+  menuIconMobile, menuLinks, openMenu, closeMenu,
+} from './menu.js';
+
+menuIconMobile.addEventListener('click', openMenu);
+menuLinks.forEach((menuLink) => {
+menuLink.addEventListener('click', closeMenu);
+});
```

Comment:
- When I click any of the mobile menu options, the correct part of the page is not displayed correctly the blue background stay and I am not able to see please address that
<img width="607" alt="Screen Shot 2022-11-23 at 22 06 08" src="https://user-images.githubusercontent.com/44745855/203637145-62564648-09d6-4f9c-a7b7-aa722c18e5e5.png">

Reply:
- [ ] Please add the close `(X)` button, and the mobile menu disappears once is clicked.